articles/pre-deployment/tests/required.md

```diff
@@ -5,22 +5,29 @@ topics:
     - pre-deployments-tests
     - tests
     - requirements
+    - production-checks
+contentType: reference
 ---
 
-# Pre-Deployment Tests: Required Fixes
+# Production Checks: Required Fixes
 
-The following tests check to see if you've completed all requirements for successful deployment to Production.
+The following checks see if you've completed all requirements for successful deployment to Production.
 
 ::: note
-See [How to Read Your Results Set](/pre-deployment/how-to-run-test#how-to-read-your-results-set) for additional information on your testing output.
+See [How to Read Your Results Set](/pre-deployment/how-to-run-test#how-to-read-your-results-set) for additional information on your checks output.
```

Comment:
This note is both on required and recommended fixes, but not on best practices https://auth0-docs-content-pr-6332.herokuapp.com/docs/pre-deployment/tests/best-practice

Is this intentional?

Reply:
It looks like its a search and replace thing.

Reply:
Best Practices checks are not automated, so there is no checks output. You have to run them manually.

Reply:
cool, I didn't know that, thanks @rkhoriander!